zio/zngio/scanner.go

```diff
@@ -58,8 +61,17 @@ func (s *zngScanner) Pull() (zbuf.Batch, error) {
 }
 
 func (s *zngScanner) scanUncompressed() ([]*zng.Record, error) {
+	uncompressed := s.reader.uncompressed
+	if bf := s.bufferFilter; bf != nil && !bf.Eval(uncompressed.Bytes()) {
```

Comment:
```suggestion
	if s.bufferFilter != nil && !s.bufferFilter.Eval(uncompressed.Bytes()) {
```

Reply:
Done in 8a59d3a0818607efc8b3ca16635d66e6ad08e5a4.